lib/assessment.md

```diff
@@ -41,10 +54,32 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+in html: in a tag you write, ex <div class="big">
+in css:
+.big {
+property: value;
+property: values;
+}
+
+#contentPane {
+height: 100px;
+width: 100px;
+}
+
+.important, .red {
+property: value;
+property: values;
+height: 100px;
+width: 100px;
+}
 
+div.p {
```

Comment:
No, this is not correct. To select `p` elements whose parents are `div` elements, use the selector `div > p`.